local_modules/qwant-maps-common/src/place_category_name.js

```diff
@@ -0,0 +1,257 @@
+  paragliding: () => _('paragliding'),
+  park: () => _('park'),
+  parking: () => _('car park'),
+  pastry: () => _('pastry shop'),
+  perfumery: () => _('perfumery'),
+  pet: () => _('pet store'),
+  pharmacy: () => _('pharmacy'),
+  photo: () => _('photo shop'),
+  picnic_site: () => _('picnic site'),
+  place_of_worship: () => _('place of worship'),
+  plumber: () => _('plumber'),
+  police: () => _('police'),
+  polling_station: () => _('polling station'),
+  post_office: () => _('post office'),
+  prison: () => _('prison'),
+  pub: () => _('pub'),
+  public_building: () => _('public building'),
+  rc_car: () => _('rc car'),
+  recycling: () => _('recycling'),
+  reservoir: () => _('reservoir'),
+  restaurant: () => _('restaurant'),
+  rowing: () => _('rowing'),
+  ruins: () => _('ruins'),
+  running: () => _('running'),
+  sailing: () => _('sailing'),
+  school: () => _('school'),
+  scuba_diving: () => _('scuba diving'),
+  seafood: () => _('fishmonger'),
+  second_hand: () => _('resale shop'),
+  shelter: () => _('shelter'),
+  shoemaker: () => _('shoemaker'),
+  shoes: () => _('shoes'),
+  shooting: () => _('shooting'),
+  skateboard: () => _('skateboard'),
+  skating: () => _('skating'),
+  skiing: () => _('skiing'),
+  soccer: () => _('soccer'),
+  sports: () => _('sports shop'),
+  sports_centre: () => _('sports centre'),
+  stadium: () => _('stadium'),
+  station: () => _('station'),
+  stationery: () => _('stationery shop'),
+  subway: () => _('subway'),
+  supermarket: () => _('supermarket'),
+  swimming: () => _('swimming pool'), // leisure=sports_centre + sport=swimming
+  swimming_area: () => _('swimming area'),
+  swimming_pool: () => _('swimming pool equipment'), // shop=swimming_pool
+  table_tennis: () => _('table tennis'),
+  tailor: () => _('tailor'),
+  tattoo: () => _('tattoo salon'),
+  tennis: () => _('tennis'),
+  theatre: () => _('theatre'),
+  theme_park: () => _('theme park'),
+  ticket: () => _('ticket shop'),
+  tobacco: () => _('tobacco shop'),
+  toll_booth: () => _('toll booth'),
+  townhall: () => _('townhall'),
+  toys: () => _('toys shop'),
+  tram_stop: () => _('tram stop'),
+  travel_agency: () => _('travel agency'),
+  university: () => _('university'),
+  variety_store: () => _('variety store'),
+  vehicle_inspection: () => _('vehicle inspection'),
+  veterinary: () => _('veterinary'),
+  video: () => _('videos shop'),
+  video_games: () => _('video games shop'),
+  viewpoint: () => _('viewpoint'),
+  watches: () => _('watches shop'),
+  water_park: () => _('water park'),
+  weapons: () => _('weapons shop'),
+  wholesale: () => _('warehouse club'),
+  wine: () => _('wine shop'),
+  winery: () => _('winery'),
+  winter_sports: () => _('winter sports resort'),
+  yoga: () => _('yoga'),
+  zoo: () => _('zoo'),
+
+  city: () => _('city'),
+  country: () => _('country'),
+  address: () => _('address'),
+  street: () => _('street'),
+};
+
+function getPlaceCategoryName({ subclass }, lang) {
```

Comment:
Not a big deal, but we could either sur params (subclass and lang), or an object as param

Reply:
The idea behind this separation is that the first parameter represents a place object. For now we only use the `subclass` attribute on it to determine the category string, but it may change in the future with other properties used.

Reply:
ok it makes sense :)